#!/usr/bin/env python2
PACKAGE = "ynoise_filter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("sizeX", int_t, 0, "pixel array size (x)", 346, 1, 1000)
gen.add("sizeY", int_t, 0, "pixel array size (y)", 260, 1, 1000)

gen.add("deltaT", int_t, 0, "Time window in micro", 10000, 1, 1000000)
gen.add("lParam", int_t, 0, "window size", 3, 3, 15)
gen.add("threshold", int_t, 0, "cutoff threshold", 2, 0, 1000)

exit(gen.generate(PACKAGE, "ynoise_filter", "ynoiseCfg"))
